zerver/migrations/0699_stream_subscriber_count_data_migration.py

```diff
@@ -0,0 +1,57 @@
+from collections import defaultdict
+
+from django.db import connection, migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from psycopg2 import sql
+
+
+def set_stream_subscribe_count(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    Stream = apps.get_model("zerver", "Stream")
+    Subscription = apps.get_model("zerver", "subscription")
+
+    stream_subsriptions = Subscription.objects.filter(
+        recipient__type=2,
+        recipient__type_id__in=Stream.objects.values("id"),
```

Comment:
What does this line do -- does it help the query plan somehow? Logically, it should be implied by the `type=2` part.

Reply:
Yes, my bad

Reply:
What we do need to filter by here is  both `active=True` and `is_user_active=True`, because we want only `active` subscriptions whose members are also active i.e. `is_user_active` = True.

Reply:
Correct.